xcube/webapi/ows/stac/context.py

```diff
@@ -33,6 +33,15 @@ def __init__(self, server_ctx: Context):
         self._datasets_ctx = server_ctx.get_api_ctx('datasets',
                                                     cls=DatasetsContext)
 
+        # TODO: decide what else to include in this list.
+        #  Listing all 11396 CRSs currently known to pyproj seems
+        #  impractical. Make it a configuration option, perhaps?
+        self._available_crss = ['OGC:CRS84', 'EPSG:4326']
```

Comment:
Is not even trying to list them and instead just validating what is handed in an option?

Reply:
That is effectively the current behaviour. Requests aren't validated against this list, and the server will attempt to parse and use whatever CRS specifier the client provides.